### PR TITLE
Adding artnews tokens for gutenberg support.

### PR DESCRIPTION
**Pull request by richhaynie**
Added missing tokens for artinamerica to support pmc-gutenberg plugin and the hubs.

### Make sure you complete these items:

- [x] Updated root CHANGELOG.md with summary of changes under `Unpublished` section
- [x] `npm run prod` in this repo outputs expected changes (excepting the issue with re-ordered partials in larva-css algorithms partials - see [LRVA-1885](https://jira.pmcdev.io/browse/LRVA-1885))

### CODE REVIEW

**Review by jameswburke — APPROVED** [review]
From what I can tell, everything looks like configuration mapping and files (mostly), so I'm going to give that a big 👍  if you're thinking it looks good.

My only concern (which might come from ignorance) is committing the `package-lock.json` files. Those diffs look unusually large to me. I had an issue last week where I was using the wrong node version, and the result was similar. I'd suggest double checking that, and/or confirming these suspicions with @laras126.

**Comment by laras126** [comment]
@richhaynie Can you update the title of this PR to describe the feature so that it shows up clearly in the commit history?

**Comment by richhaynie** [comment]
@jameswburke or @laras126 which version of node should I be running for this?

**Comment by laras126** [comment]
@richhaynie Maybe you figured it out - run `nvm use` to reference the recommended version (which is 14 - and you'd need to [install nvm](https://github.com/nvm-sh/nvm) if you haven't before).

File: package.json [hunk 1]
```diff
@@ -75,6 +75,7 @@
     "gulp-sass": "^4.1.0",
     "gulp-stylelint": "^13.0.0",
     "jest": "^24.8.0",
+    "larva": "0.0.4",
```
Comment (laras126) [inline package.json:78]: This looks fishy...

**Review by laras126 — APPROVED** [review]
Looks good!

**Pull request merged**